### PR TITLE
display region in workspace combo box

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
- display region in workspace combo box
- do not support to create custom workspace
- fix build error of eclipse

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-applicationinsights/src/main/java/com/microsoft/azure/toolkit/intellij/applicationinsights/creation/WorkspaceComboBox.java [hunk 1]
```diff
@@ -57,7 +62,7 @@ protected LogAnalyticsWorkspaceConfig doGetDefaultValue() {
     @Override
     public void setValue(LogAnalyticsWorkspaceConfig val) {
         if (Objects.nonNull(val) && val.isNewCreate()) {
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-applicationinsights/src/main/java/com/microsoft/azure/toolkit/intellij/applicationinsights/creation/WorkspaceComboBox.java:65]: check equals() of log workspace config class

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-applicationinsights/src/main/java/com/microsoft/azure/toolkit/intellij/applicationinsights/creation/WorkspaceComboBox.java [hunk 2]
```diff
@@ -61,11 +61,11 @@ protected LogAnalyticsWorkspaceConfig doGetDefaultValue() {
 
     @Override
     public void setValue(LogAnalyticsWorkspaceConfig val) {
+        this.draftItems.clear();
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-applicationinsights/src/main/java/com/microsoft/azure/toolkit/intellij/applicationinsights/creation/WorkspaceComboBox.java:64]: clear items() when region changed

Comment (ivywei0125) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-applicationinsights/src/main/java/com/microsoft/azure/toolkit/intellij/applicationinsights/creation/WorkspaceComboBox.java:64]: should not clear items in set value

**Review by wangmingliang-ms — APPROVED** [review]
remember to resolve comments in later PRs.

**Pull request merged**